src/man/sssd.conf.5.xml

```diff
@@ -202,6 +202,11 @@
                             <para>
                                 Comma separated list of services that are
                                 started when sssd itself starts.
+                                <phrase condition="have_systemd">
+                                    or SSSD 1.15.0 or later the services' list is optional
```

Comment:
In case we're not backporting those patches the version number doesn't make sense.
And there's a typo there as well :-\